app/content/deployment-options/custom-deployment/operations.md

```diff
@@ -1,5 +1,49 @@
 ## Operations
 
+### Configuring proxy
+
+When the instance is going to be installed behind a `proxy` it is necessary to:
```

Comment:
I would just remove the code format for the word "proxy".

Reply:
Done!

Thanks! 🙌